src/main/java/com/revature/Flumblr/controllers/PostController.java

```diff
@@ -141,11 +141,31 @@ public ResponseEntity<PostResponse> getPost(@PathVariable String postId,
     public ResponseEntity<?> commentOnPost(@RequestBody NewCommentRequest req,
             @RequestHeader("Authorization") String token) {
 
-        tokenService.validateToken(token, req.getUser_id());
+        tokenService.validateToken(token, req.getUserId());
```

Comment:
You replaced one instance of validateToken with extractUserId? Are we not replacing all instances with it?